src/Symfony/Component/AssetMapper/ImportMap/ImportMapManager.php

```diff
@@ -295,13 +295,15 @@ private function requirePackages(array $packagesToRequire, ImportMapEntries $imp
         $resolvedPackages = $this->resolver->resolvePackages($packagesToRequire);
         foreach ($resolvedPackages as $resolvedPackage) {
             $importName = $resolvedPackage->requireOptions->importName ?: $resolvedPackage->requireOptions->packageName;
-
+            [$packageName, $filePath] = ImportMapConfigReader::splitPackageNameAndFilePath($importName);
```

Comment:
```suggestion
            [$packageName, $filePath] = ImportMapConfigReader::splitPackageNameAndFilePath($resolvedPackage->requireOptions->packageName;);
```

This is a bit odd - it reflects why the two new properties were needed. When you use `importmap:require`, you can require a package but give it a different import name - e.g. `vue/dist/vue.esm-bundler.js=vue`. So, the `->packageName` here represents the "package + path" from that - `vue/dist/vue.esm-bundle.js`.

The fact that the import name (i.e. key) could possibly not equal the package+path for an entry in `importmap.php` is a bit of a problem now that we just store the `version` there instead of the full CDN path. Probably, when writing the `importmap.php`, if `importName != packageName / filePath`, then we should output a new `package` key that contains `packageName + filePath` (I wish I could think of a better name for this packageName + filePath combination :) ). 

Anyway, that can be a different PR - you can work on it if you'd like, but it's also on my list - I'm mostly thinking out loud.

Reply:
FYI - I'm working on the PR to make the `packageName + filePath` stuff a bit clearer, as it affects many places.